data/scripts/spells/monster/gaz'haragoth_summon.lua

```diff
@@ -1,31 +1,44 @@
+dofile("data/scripts/spells/monster/gaz_functions.lua")
+
 local spell = Spell("instant")
 
 function spell.onCastSpell(creature, var)
-local t, spectator = Game.getSpectators(creature:getPosition(), false, false, 5, 5, 5, 5)
+    local t, spectator = Game.getSpectators(creature:getPosition(), false, false, 25, 25, 25, 25)
     local check = 0
     if #t ~= nil then
         for i = 1, #t do
-		spectator = t[i]
-            if spectator:getName() == "Minion Of Gaz'haragoth" then
-               check = check + 1
+            spectator = t[i]
+            if spectator:getName() == "Minion of Gaz'haragoth" then
+                check = check + 1
+            end
+        end
+    end
+
+    if (check >= GazVariables.MaxSummons) then
+        return false
+    else
+        if (check < GazVariables.MinionsNow) then
+            for i = 1, (GazVariables.MinionsNow - check) do
+                local monster = Game.createMonster("Minion of Gaz'haragoth", creature:getPosition(), true, false)
```

Comment:
unused variable 'monster'

Reply:
The location is inside the for, so it's complaining, it needs to be declared outside and modified inside.